<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureProjects</class>
 <widget class="QDialog" name="ConfigureProjects">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>378</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configure tools</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="projects">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <widget class="QPushButton" name="projectsNew">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="projectsDelete">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>4</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="generalTab">
          <attribute name="title">
           <string>General</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLineEdit" name="name"/>
               </item>
               <item>
                <widget class="QLabel" name="nameStatus">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>OK</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Path</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLineEdit" name="path"/>
               </item>
               <item>
                <widget class="QPushButton" name="pathBrowse">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Defines</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="defines"/>
             </item>
             <item row="3" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Include</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="3" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QListWidget" name="includes">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>80</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="includesAdd">
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="includesDelete">
                   <property name="text">
                    <string>Delete</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>43</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="clangTab">
          <attribute name="title">
           <string>Clang</string>
          </attribute>
          <widget class="QCheckBox" name="clangCompiler">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>211</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Clang compiler warnings</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="clangAnalyser">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>191</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Clang analysis</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="cppcheckTab">
          <attribute name="title">
           <string>Cppcheck</string>
          </attribute>
          <widget class="QCheckBox" name="cppcheckEnabled">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>141</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Cppcheck</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="gccTab">
          <attribute name="title">
           <string>Gcc</string>
          </attribute>
          <widget class="QCheckBox" name="gccEnabled">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>201</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Compiler warnings</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>includesAdd</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>newInclude()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>includesDelete</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>deleteInclude()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectsNew</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>newProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectsDelete</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>deleteProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projects</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>selectProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathBrowse</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>pathBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clangCompiler</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>saveCompilerSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clangAnalyser</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>saveCompilerSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cppcheckEnabled</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>saveCompilerSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gccEnabled</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>saveCompilerSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>name</sender>
   <signal>editingFinished()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>path</sender>
   <signal>editingFinished()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defines</sender>
   <signal>editingFinished()</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigureProjects</receiver>
   <slot>nameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pathBrowse()</slot>
  <slot>newProject()</slot>
  <slot>deleteProject()</slot>
  <slot>newInclude()</slot>
  <slot>deleteInclude()</slot>
  <slot>selectProject()</slot>
  <slot>apply()</slot>
  <slot>saveCompilerSettings()</slot>
  <slot>nameChanged(QString)</slot>
 </slots>
</ui>
